content/authors/hpatro.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Harkrishn Patro
+extra:
+    photo: '/assets/media/authors/hpatro.png'
+    github: hpatro
+---
+
+Harkrishn Patro is a Software Engineer at AWS.
+
+Harkrishn is passionate about distributed systems, databases, anything related to data.
```

Comment:
Any thing you like outside of work? Badminton, surfing, making hats, etc.?

Reply:
Somehow you guess my favorite sport to play. Will add some :)